src/js/messaging/components/compose/MessageAttachments.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { formatFileSize } from '../../utils/helpers';
+
+import MessageUploadedAttachment from './MessageUploadedAttachment';
+
+class MessageAttachments extends React.Component {
+  render() {
+    const cssClass = classNames(
+      'msg-attachments',
+      this.props.cssClass
+    );
+
+    const files = this.props.files.map((file, index) => {
+      const fileSize = formatFileSize(file.size);
+      return (
+        <li key={index}>
```

Comment:
Using index as a React key can result in some wonky behavior, so it's better to use a more unique identifier. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318#.jjbrw9qo2